server/routes/river-and-sea-levels.js

```diff
@@ -43,10 +79,21 @@ module.exports = [{
   method: 'POST',
   path: '/river-and-sea-levels',
   handler: async (request, h) => {
-    const { location } = request.payload
-    if (typeof location === 'undefined' || location === '') {
+    const { location, river } = request.payload
```

Comment:
I think you can change this to `const { location, river, 'search-type': searchType } = request.payload` just tested it using:
```
const { 'test-hyphen': testHyphen } = { 'test-hyphen': 'test' }

console.log(testHyphen)
```